src/bin/alert_worker.rs

```diff
@@ -1,10 +1,19 @@
 use boom::{alert, conf, types::ztf_alert_schema, worker_util};
+use clap::Parser;
 use redis::AsyncCommands;
-use std::env;
 use std::sync::{Arc, Mutex};
 use tracing::{error, info, warn, Level};
 use tracing_subscriber::FmtSubscriber;
 
+#[derive(Parser)]
+struct Cli {
+    #[arg(required = true, help = "Name of stream to ingest")]
+    stream: Option<String>,
+
+    #[arg(long, value_name = "FILE", help = "Path to the configuration file")]
```

Comment:
what does `value_name` do?

Reply:
That's for when you type `--help`, so it shows `--config <FILE>`.

Reply:
got it, thanks!